.gitignore

```diff
@@ -208,11 +208,6 @@ tools/swig/test/Array.py
 
 # SIMD generated files #
 ###################################
-# config headers of dispatchable sources
-*.dispatch.h
-# wrapped sources of dispatched targets, e.g. *.dispatch.avx2.c
-*.dispatch.*.c
-*.dispatch.*.cpp
```

Comment:
Should we also move the files below to the build directory?

Reply:
We're going to remove the duplicate build setup as soon as possible, so is there a benefit to spending time on this now? Seems like we can merge this and move on, given that the immediate problem is solved here?